pom.xml

```diff
@@ -355,6 +355,11 @@
 <!--            <version>1.7.30</version>-->
 <!--            <scope>test</scope>-->
 <!--        </dependency>-->
+        <dependency>

```

Comment:
is this necessary?

Reply:
yes, it fixes the problem of the docker container not starting in testcontainers